reactos/drivers/usb/usbhub_new/usbhub.c

```diff
@@ -3347,86 +3329,75 @@ USBH_FdoIdleNotificationCallback(IN PVOID Context)
                           FALSE,
                           NULL);
 
-    Port = 0;
+    PortData = HubExtension->PortData;
 
-    if (HubExtension->HubDescriptor->bNumberOfPorts == 0)
+    for (Port = 0;
+         Port < HubExtension->HubDescriptor->bNumberOfPorts;
+         Port++)
     {
         if ((HubExtension->HubFlags & USBHUB_FDO_FLAG_DEVICE_STOPPING) ||
```

Comment:
usbhub.c:3338: before the latest change, this USBHUB_FDO_FLAG_DEVICE_STOPPING/CheckIdleAbort check did not happen on every loop. I think it should probably be removed?